src/Kiota.Builder/Refiners/CommonLanguageRefiner.cs

```diff
@@ -263,12 +263,12 @@ private static void ReplaceReservedParameterNamesTypes(CodeMethod currentMethod,
                 var shouldInsertUsing = false;
                 if(BinaryType.Equals(currentMethod.ReturnType?.Name)) {
                     currentMethod.ReturnType.Name = symbol;
-                    shouldInsertUsing = true;
+                    shouldInsertUsing = string.IsNullOrWhiteSpace(ns);
```

Comment:
Added this to avoid generating any import. However, I continued getting blank imports in my TS generation. What is the best way to achieve this?

Reply:
I think the condition should be negated to start with.

```suggestion
                    shouldInsertUsing = !string.IsNullOrWhiteSpace(ns);
```

Reply:
Yeah. That worked.

Reply:
thanks for confirming, resolving